javascript/app-model-exposure/src/utils/html.ts

```diff
@@ -0,0 +1,7 @@
+/*!
+ * Copyright (c) Microsoft. All rights reserved.
+ * Licensed under the MIT license. See LICENSE file in the project.
+ */
+export function returnValidHTMLId(id: string): string {
```

Comment:
can we rename this function? `makeHtmlId` perhaps? `return` is implicit with most functions

Reply:
Makes sense, thanks